proposed/project-file-simplification.md

```diff
@@ -0,0 +1,351 @@
+
+When a there is a breaking change or a new warning, there is a new MSBuild property (as today). For breaking changes or new warnings, the feature will be turned off if it `false` or the equivalent `disable`. The defaults specified by `Base` will turn on the features we strongly recommend. If you disagree with this for your project, you can reset the value to `false`.
+
+The key element of the `Base` design is that it is versioned and released with each SDK. This allows you to initially experiment with, and then adopt all the breaking changes and new warnings with a single value. The proposed structure of this version is the corresponding SDK version number, such as `10.0.100`, with a possible simplification to just `10`. The `Base` defaults include the `TargetFramework` so the user manages a single version number, in most cases.
+
+However, key to this design is that `Base` is not required to match either your SDK or your `TargetFramework`. For example, if you use an LTS strategy and want to ensure you get the latest warnings to benefit your code and prepare for future upgrade, you might have:
+
+```XML
+<Project Base="11.0.100" 
+         Type="Console"> 
+   <PropertyGroup>
+        <TargetFramework>net10<TargetFramework>
+   </PropertyGroup>
+</Project> 
+```
+
+Similarly, you can use `Base` to get the warnings applicable to you if you are running a .NET Framework or .NET Standard project where the `TargetFramework` does not change.
+
+When upgrading to adopt breaking changes and new warnings, the user can just change the `Base` value, or they can use tooling with info on the new behavior.
+
+This design considers both very small and very large scenarios. For small projects and new users, the project file is simple and contains only two attributes that they did not enter. Both of these attributes have names/values they understand.
+
+For large repos with many projects, we embrace `Imports` files and look forward to feedback to drive further refinements.
+
+### Separately controlling `TargetFramework` and tooling behavior
+
+For many users, upgrading their tooling breaking changes/warnings along with their `TargetFramework` keeps life simple:
+
+```XML
+<Project Base="10.0.100" 
+         Type="Console"> 
+</Project> 
+```
+
+For users that wish to postpone or avoid breaking changes/warnings, they can use a lower version of tooling defaults:
+
+```XML
+<Project Base="10.0.100" 
+         Type="Console">
+    <PropertyGroup>
+        <TargetFramework>net11</TargetFramework>
+    </PropertyGroup>
+</Project> 
+```
+
+For users that are not ready to upgrade their `TargetFramework` but want any new warnings that may improve their application, they can use a higher version of tooling defaults:
```

Comment:
The `Base` version is the same in all three examples here. Did you mean for it to change in this example (based on this text)?

Reply:
I believe the text is referring to the fact that `10.0.100` is a higher tooling version than the tfm of `net482`?